configs/coderello_docs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "index_name": "coderello_docs",
+  "start_urls": [],
```

Comment:
Please keep start_urls so that we have a better idea of what are the documentation pages you want to index

Reply:
Could it be the link to the main page of the website?

Reply:
I thought that if I the config file doesn't have `start_urls` then the parser will use the `sitemap.xml`. Did I get it wrong?

Reply:
> Could it be the link to the main page of the website?

Yes that's correct, however you will need to remove the whole .start_url key. Anyway, please use the main page of the website. 

You can use   `"scrap_start_urls": false`, to avoid indexing the first page